spec/rails_spec.rb

```diff
@@ -271,13 +279,25 @@ class Fixnum
 
           expect(response.header['Per-Page']).to eq(
             case ApiPagination.config.paginator
-            when :pagy          then Pagy::VARS[:items].to_s
```

Comment:
You'll need to add this back in (and also add it to your new spec below!), probably got lost in a merge conflict!